README.md

```diff
@@ -85,932 +119,667 @@ Unlike many authorization systems where a service controls access to resources i
+        SignatureBytes["Signature (raw bytes)"]
+      
+        subgraph SigPayload ["Signature Payload"]
+            VarsigHeader["Varsig Header"]
+
+            subgraph UcanPayload ["Token Payload"]
+                fields["..."]
+            end
+        end
+    end
+```
 
-Any issuer of a UCAN MAY later revoke that UCAN or the capabilities that have been derived from it further downstream in a proof chain.
+For example:
 
-This mechanism is eventually consistent and SHOULD be considered the last line of defense against abuse. Proactive expiry via time bounds or other constraints SHOULD be preferred, as they do not require learning more information than what would be available on an offline computer.
+``` js
+[
+  {"/": {"bytes": "7aEDQLYvb3lygk9yvAbk0OZD0q+iF9c3+wpZC4YlFThkiNShcVriobPFr/wl3akjM18VvIv/Zw2LtA4uUmB5m8PWEAU"}},
+  {
+    "h": {"/": {"bytes": "NBIFEgEAcQ"}},
+    "ucan/example@1.0.0-rc.1": {
+      "hello": "world"
+    }
+  }
+]
+```
 
-While some resources are centralized (e.g. access to a server), others are unbound from specific locations (e.g. a CRDT), in which case it will take longer for the revocation to propagate.
+### Payload
 
-Every resource type SHOULD have a canonical location where its revocations are kept. This list is non-exclusive, and revocation messages MAY be gossiped between peers in a network to share this information more quickly.
+A UCAN's Payload MUST contain at least the following fields:
 
-It is RECOMMENDED that the canonical revocation store be kept as close to (or inside) the resource it is about as possible. For example, the WebNative File System maintains a Merkle tree of revoked CIDs at a well-known path. Another example is that a centralized server could have an endpoint that lists the revoked UCANs by [canonical CID].
+| Field   | Type                                      | Required | Description                                                 |
+|---------|-------------------------------------------|----------|-------------------------------------------------------------|
+| `iss`   | `DID`                                     | Yes      | Issuer DID (sender)                                         |
+| `aud`   | `DID`                                     | Yes      | Audience DID (receiver)                                     |
+| `sub`   | `DID`                                     | Yes      | Principal that the chain is about (the [Subject])           |
+| `cmd`   | `String`                                  | Yes      | The [Command] to eventually invoke                          |
+| `args`  | `{String : Any}`                          | Yes      | Any [Arguments] that MUST be present in the Invocation      |
+| `nonce` | `Bytes`                                   | Yes      | Nonce                                                       |
+| `meta`  | `{String : Any}`                          | No       | [Meta] (asserted, signed data) — is not delegated authority |
```

Comment:
Maybe break out the non-required fields into a separate table below, since all of this is under MUST CONTAIN?

And throw `iat` in there, too?

Reply:
The rationale is that `iat` is not in the delegation spec since it's not particularly helpful for delegation. Invocation does have an `iat`, so it's not a common field across all UCAN formats